src/tools/extract-conditionals.cpp

```diff
@@ -0,0 +1,279 @@
+#include "extractor/opening_hours.hpp"
+#include "util/log.hpp"
+
+#include <boost/geometry.hpp>
+#include <boost/geometry/index/rtree.hpp>
+
+#include <osmium/io/any_input.hpp>
+#include <osmium/tags/regex_filter.hpp>
+
+#include <shapefil.h>
+
+#include <fstream>
+#include <iostream>
+#include <unordered_map>
+
+#include <cstdlib>
+#include <ctime>
+
+// better place in third_party/libosmium/include/osmium/tags/regex_filter.hpp
```

Comment:
Maybe if you ping @joto and be nice about it he might include it in libosmium ;)